### PR TITLE
[tests-only][full-ci] backport e2e runner implementation

### PR DESCRIPTION
**Pull request by PrajwolAmatya**
### Description
This PR backports the e2e runner implementation from PR: https://github.com/owncloud/ocis/pull/8736 and PR https://github.com/owncloud/ocis/pull/8739

Part of: https://github.com/owncloud/ocis/issues/8794

### CODE REVIEW

File: .drone.star [hunk 1]
```diff
@@ -140,7 +140,16 @@ config = {
         "skipExceptParts": [],
     },
     "e2eTests": {
-        "skip": False,
+        "basic": {
+            "skip": False,
+            "totalParts": 3,  # divide and run all suites in parts (divide pipelines)
```
Comment (saw-jan) [inline .drone.star:145]: Also port PR https://github.com/owncloud/ocis/pull/8739

**Review by saw-jan — APPROVED** [review]
LGTM :+1:

**Review by SagarGi — APPROVED** [review]

**Comment by PrajwolAmatya** [comment]
Currently, the CI has failed since it needs the latest web bump on ocis stable after the fixes from this PR https://github.com/owncloud/web/pull/10813. So, moving this PR to blocked.

**Comment by saw-jan** [comment]
Will there be another patch release of web-8?

CC @kulmann @JammingBen

**Comment by kulmann** [comment]
> Will there be another patch release of web-8?
> 
> CC @kulmann @JammingBen

yes

**Comment by saw-jan** [comment]
Can be done after https://github.com/owncloud/ocis/pull/9153 is merged

**Pull request merged**